src/components/GGanttChart.vue

```diff
@@ -135,10 +154,12 @@ const getChartRows = () => {
   }
   defaultSlot.forEach((child) => {
     if (child.props?.bars) {
+      console.log(child)
```

Comment:
We will have to make sure to remove any loggings before we merge this.

Reply:
removed